src/imageprocessing.jl

```diff
@@ -0,0 +1,113 @@
+using Images, ImageFiltering
+using FileIO
+using StatsBase
+
+"""
+    images_to_maps(...)
+
+* has nearly the same inputs as convert_image_to_world but there is a directorypath instead of a filepath of one image and
+* a targetpath, where you save the 12 maps 
+"""
+function images_to_maps(dirsourcepath, targetpath = joinpath(@__DIR__, "..","input","world","monthly_maps"),transpose_image = true,  blurred=1, imglong=5400, imglat=2700, nlongitude=128, limit_ocean = 10, limit_land= 190, limit_seaice=205 )
+    
+    if isdir(dirsourcepath)
+        for (root, dirs, files) in walkdir(dirsourcepath)
+            for file in files
+                if occursin("world", string(file))
+                    filepath = joinpath(@__DIR__, root, file) 
+                    world = convert_image_to_world(filepath, transpose_image, blurred, imglong, imglat, nlongitude, limit_ocean, limit_land, limit_seaice)
+                    save_world_by_month(world, string(file), targetpath)
+                end
+            end
+        end
+    end
+end 
+
+"""
+    convert_image_to_world(...)
+
+* imagefile is the filapath of that lanscape we want to convert into our map with dicrete values for each landcover
+* transpose_image is nessessary because without transposing each image from NASA website, we would create a 
+    symmetrically mirrored map
+* imglong and imglat are the longitude and latitude of the input image
+* nlongitude is the preferred longitude for the nearest neighbor interpolation
+* limit_[...] is the limiting value for the classification from grayscale per landcover, 
+    0 <= ocean (5) <= limit_ocean <= land (1) <= limit_land <= ice (2) <= limit_seaice <= snow (3) <= 255
+"""
+function convert_image_to_world(imagefile, transpose_image=true, blurred=1, imglong=5400, imglat=2700, nlongitude=128, limit_ocean = 10, limit_land= 190, limit_seaice=210)
```

Comment:
The variables `imglong` and `imglat` are not used anywhere. Please remove them from here and from `images_to_maps`